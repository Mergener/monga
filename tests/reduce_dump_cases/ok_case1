function hello(a : int, b : float) {
	print(4);
}

var test : int;