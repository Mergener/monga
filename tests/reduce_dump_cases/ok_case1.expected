type r1
parameter r1
parameters r1
type r1
parameter r1
parameters r2
opt_parameters r1
opt_ret_type r1
numeral r2
exp_primary r2
exp_postfix r1
exp_unary r1
exp_multiplicative r1
exp_additive r1
exp_conditional r1
exp r1
exps r1
opt_exps r2
call r1
block r5
statements r1
opt_statements r2
block r1
def_function r1
definition r2
definitions r1
type r1
def_variable r1
definition r1
definitions r2
module r1
